src/tinypeel/tinypeel.py

```diff
@@ -91,6 +91,7 @@ def peelingCycle(pedigree, peelingInfo, args, singleLocusMode=False):
 # right now the posterior term is updated for all of the sires/dams no matter
 # whether or not they have been changed since the last update.
 
+def updatePosterior(pedigree, peelingInfo, sires, dams) :
```

Comment:
@XingerTang are these changes for `AlphaPeel` repo or for `tinyhouse` repo?

Reply:
Similarly here, the ``pre-commit`` ignores all the submodules.